docs/metrics.md

```diff
@@ -114,28 +114,28 @@ Sum of VMIs per phase and node.
 `phase` can be one of the following: [`Pending`, `Scheduling`, `Scheduled`, `Running`, `Succeeded`, `Failed`, `Unknown`]. Type: Gauge.
 
 ### kubevirt_vmi_storage_flush_requests_total
-Storage flush requests. Type: Counter.
+Total storage flush requests. Type: Counter.
 
 ### kubevirt_vmi_storage_flush_times_ms_total
 Total time (ms) spent on cache flushing. Type: Counter.
 
 ### kubevirt_vmi_storage_iops_read_total
-I/O read operations. Type: Counter.
+Number of I/O read operations. Type: Counter.
```

Comment:
The word "Total" was added to many descriptions but not this one and others. Feels a bit incoherent.